### DISCUSSION TITLE
From where can I enable RSS for updates?

### DISCUSSION THREAD

**Discussion by cx48** [timlrx/tailwind-nextjs-starter-blog#994] [General]
I want a RSS link for updates to my blogs. How can I enable RSS>

**Comment by ray-android** [comment]
There is an RSS feature. You can use the URL format, e.g., https://tailwind-nextjs-starter-blog.vercel.app/feed.xml and https://tailwind-nextjs-starter-blog.vercel.app/tags/next-js/feed.xml.

For coding, you can refer to scripts/rss.mjs.

**Reply by cx48** [reply]
> There is an RSS feature. You can use the URL format, e.g., https://tailwind-nextjs-starter-blog.vercel.app/feed.xml and https://tailwind-nextjs-starter-blog.vercel.app/tags/next-js/feed.xml.
> 
> For coding, you can refer to scripts/rss.mjs.

Thank you sm for your help!
